packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.html

```diff
@@ -0,0 +1,38 @@
+<template>
+  <lightning-card title={label} class="container">
+    <lightning-button-icon
+      class="slds-p-bottom_small"
+      slot="actions"
+      onclick={toggleVisibility}
+      icon-name={collapseIcon}
+      alternative-text="toggle recent queries visibility"
+      variant="bare"
+    >
+    </lightning-button-icon>
+    
+    <template if:false={isCollapsed}>
+      <template if:false={hasQueries}>
+        <div class="slds-text-body_small empty-list-message slds-p-horizontal_medium">{labels.emptyListLabel}</div>
+      </template>
+      <ul class="slds-m-around_small slds-p-bottom_xxx-small">
+          <template for:each={queries} for:item="query" for:index="index">
+            <li
+              key={q}
```

Comment:
:thinking: Where does `q` come from? Shouldn't it be `key={query}` instead?
You might want to ensure that queries are not duplicated, otherwise the component will crash.

Reply:
oops yes it should be key={query}. forgot to change the var name in the new commit!